app/models/miq_server/worker_management/monitor/class_names.rb

```diff
@@ -62,7 +62,7 @@ module MiqServer::WorkerManagement::Monitor::ClassNames
     MiqSmartProxyWorker
     MiqSmisRefreshWorker
     MiqStorageMetricsCollectorWorker
-    MiqWebsocketWorker
+    MiqCockpitWsWorker
```

Comment:
This looks like a bug...I don't think the original worker was meant to be removed